examples/01_plotting/plot_surf_atlas.py

```diff
@@ -0,0 +1,57 @@
+"""
+Demonstrate basic loading and plotting of a cortical surface atlas.
+===================================================================
+The Destrieux parcellation (Destrieux et al, 2010) in fsaverage5 space as
+distributed with Freesurfer is used as the chosen atlas.
```

Comment:
Maybe add the references mentioned below (Dale et al, 1999, Fischl et al 1999) here after Freesurfer